README.md

```diff
@@ -1,25 +1,39 @@
 # GyGO: an E-commerce Video Object Segmentation Dataset by Visualead
-## (Coming soon...)
-
 ---
 
 **To get updates as we continually release the dataset, add this project to your "watch" list in github**
 
-GyGO (pronounced jai-go), which we will release in parts over the following weeks, is a dataset that is focused on a specific, simple use case for video object segmentation: e-commerce. 
+## Introduction
+GyGO (pronounced jai-go), which release in parts over the following weeks, is a dataset that is focused on a specific, simple use case for video object segmentation: e-commerce.
 It will eventually consist of hundreds of short videos and was partly inspired by the release of the DAVIS datasets (link below).
 
-On the one hand, the sequences are quite simple in that they are virtually devoid of occlusions, fast motions or many of the other complexity inducing attributes mentioned above. 
+On the one hand, the sequences are quite simple in that they are virtually devoid of occlusions, fast motions or many of the other complexity inducing attributes mentioned above.
 On the other hand, the objects in these videos vary wildly in their "semantic categories": toys, cloths, models and office stuff.
 The GyGO dataset specializes in smartphone captured videos and its frames are relatively sparse (the annotated video speed is ~5 fps).
 
 We release it publicly with two goals in mind:
-- There is a severe lack of data in the space of video object segmentation at the moment. 
-With only hundreds of annotated videos, we believe every contribution has the potential to increase performance. 
+- There is a severe lack of data in the space of video object segmentation at the moment.
+With only hundreds of annotated videos, we believe every contribution has the potential to increase performance.
 In our internal (soon to be published) analysis we have shown that a joint training on the GyGO and DAVIS datasets yields improved inference results. [todo: confirm]
 - To promote a more open, sharing culture and encourage other researchers to join our efforts :) The DAVIS dataset and the research ecosystem that grew it have been massively useful for us. We hope the community will find our datasets useful as well.
 
+## Folder structure
+```
+.
+├── OriginalFrames # folders containing RGB videos
+|   ├── 480p
+|   └── originalSize 
+├── Annotations # folders containing binary annotations
+|   ├── 480p
+|   └── originalSize
+└── ImageSets
+    ├── Train.txt # contains a list of all the images for training
+    ├── trainval.txt # contains ALL the images for training and for validation    
+    └── val.txt # contains only the images for validation
+```
```

Comment:
Maybe here we could be more specific and said that the ImageSets `*.txt` files include list of sequences and not a list of images.